scripts/zones/Yuhtunga_Jungle/mobs/Carthi.lua

```diff
@@ -15,18 +15,18 @@ end;
 -----------------------------------
 
 function onMobDeath(mob, player, isKiller)
+    if (isKiller) then -- Causes this to only run once per Carthi kill
```

Comment:
I got to do my fight twice on retail coz I died and reraised, and before I stood up bio killed the 2nd chimp. I was a little po'd at the time. Woudn up having to come back later with help. NO SORO. >:(